docs/spec/spec.md

```diff
@@ -323,6 +323,7 @@ spec:  # One of "runLatest", "release", "pinned" (DEPRECATED), or "manual"
   release:
     # Ordered list of 1 or 2 revisions. First revision is traffic target
     # "current" and second revision is traffic target "candidate".
+    # It is possible to specify `"@latest"` string as a shortcut to the lastest available revision.
```

Comment:
🤔 We should add a conformance test for this.

I'm also wondering what other docs need to be updated either here of in `knative/docs`.  @dgerd Are you aware of any documentation of release mode we should update?

Reply:
Tests added

Reply:
There are not any other docs that need immediate update. Most our of documentation on using Knative is in the form of samples. Updating at least one or two samples to use this syntax would be nice.

Reply:
I can update samples, but I definitely want to do this independently of this.